fabtests/functional/rdm.c

```diff
@@ -53,6 +53,11 @@ static int run(void)
 		if (--nconn && !ret) {
 			ret = ft_accept_next_client();
 		}
+		if (oob_sock >= 0 && !opts.dst_addr) {
+			ret = ft_sock_sync(oob_sock, 1);
+			if (ret)
+				return ret;
+		}
```

Comment:
This should be moved into the `if` block above, otherwise the server would call extra sync for the first client.